indra/newview/lldrawpoolbump.cpp

```diff
@@ -78,7 +78,9 @@ static S32 cube_channel = -1;
 static S32 diffuse_channel = -1;
 static S32 bump_channel = -1;
 
-#define LL_BUMPLIST_MULTITHREADED 0 // TODO -- figure out why this doesn't work
+// Enabled after changing LLViewerTexture::mNeedsCreateTexture to an
+// LLAtomicBool; this should work just fine, now. HB
```

Comment:
It was merged into Linden Lab master branch at one point but then was absent from main branch so that's why it's dangling here ☹️. I can revert the commit but I'm not sure if that will appease the commit bot?